documentation/Installation.md

```diff
@@ -503,10 +504,23 @@ The known IDEs that support validation are:
 
 The inventory file consists of the following sections.
 
+### apiVersion
+
+API version defines the set of supported inventory sections, properties, and values.
+The `cluster.yaml` and each procedure inventory have independent versioning.
+To find currently supported `apiVersion` for the particular procedure inventory file,
+refer to the documentation about the corresponding procedure in [Provided Procedures](/documentation/Maintenance.md#provided-procedures).
+
+The following example shows the currently supported API version for the `cluster.yaml`:
+
+```yaml
+apiVersion: v1
+```
+
```

Comment:
> API version defines the set of supported inventory sections, properties, and values.

As for me, it sounds like particular API version should have a fixed/constant set of supported section/properties/values. But in fact, set of supported section/properties/values is fixed/constant only for particular Kubemarine version.

For example, in Kubemarine `v1.2.0` we add new section `containerdRegistiesConfig` (non-major change). It uses the same `apiVersion` as Kubemarine `v1.1.0` but set of sections/properties/values is different. So `apiVersion` does not define this set. It is used for something else (I am not sure for what)